modules/integration/tests/src/test/java/org/wso2/appmanager/integration/utils/AppmTestConstants.java

```diff
@@ -22,6 +22,16 @@
 
     public static final String AVERAGE = "average";
 
+    public static final String BUSINESS_OWNER_ID = "businessOwnerId";
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/e263afed698e7dbd1253c0136563513fd4051eea